src/server/saved_objects/service/lib/repository.js

```diff
@@ -35,12 +35,14 @@ export class SavedObjectsRepository {
     const {
       index,
       mappings,
+      schema,
       callCluster,
       onBeforeWrite = () => { },
     } = options;
 
     this._index = index;
     this._mappings = mappings;
```

Comment:
I think it would make sense for mappings and schema to merge at some point.

Reply:
I completely agree.